README.rst

```diff
@@ -72,6 +72,12 @@ Slack: `#go on Bazel Slack`_, `#bazel on Go Slack`_
 
 Announcements
 -------------
+2023-03-27
+  Release
+  `v0.39.0 <https://github.com/bazelbuild/rules_go/releases/tag/v0.39.0>`_
+  is now available. This release includes support for Bzlmod and various
```

Comment:
We had bzlmod before this release. Maybe just say that SDK registration has been simplified? We could also point out the new `//go` target.